### PR TITLE
W-0 catch all redirects on dev

### PR DESCRIPTION
**Pull request by caseycs**
This is how it looks: https://www.dropbox.com/s/ia8smewri28yk3c/Screenshot%202015-07-07%2016.39.08.png?dl=0

I'm open to all kinds of comments!

### CODE REVIEW

File: lib/controller/sfWebController.class.php [hunk 1]
```diff
@@ -204,4 +209,35 @@ public function redirect($url, $delay = 0, $statusCode = 302)
...
+            }
+            $url = str_replace('%f', $rowN['file'], $urlTemplate);
+            $url = str_replace('%l', $rowN['line'], $url);
+            $fileLine = "<a href=\"{$url}\">{$rowN['file']}:{$rowN['line']}</a>";
+            $html .= "<li><code>{$rowN['class']}::{$rowN['function']} in {$fileLine}</code></li>";
+        }
+        $html .= "</ul>";
+
+        //allow to hit "enter" to bypass
+        $html .= '<script type="text/javascript">document.getElementById("next").focus();</script>';
```
Comment (thijsBreker) [inline lib/controller/sfWebController.class.php]: The type attribute isn't needed

File: lib/controller/sfWebController.class.php [hunk 2]
```diff
@@ -204,4 +209,35 @@ public function redirect($url, $delay = 0, $statusCode = 302)
     $response->setContent(sprintf('<html><head><meta http-equiv="refresh" content="%d;url=%s"/></head></html>', $delay, htmlspecialchars($url, ENT_QUOTES, sfConfig::get('sf_charset'))));
     $response->send();
   }
+    
+    private function catchRedirectOnDev($url, $statusCode)
+    {
+        $html = "<p>Redirect <code>{$statusCode}</code> to <a href=\"{$url}\" id=\"next\">{$url}</a>";
+        $html .= "<br/>Click <code>Enter</code> to follow</p>";
```
Comment (thijsBreker) [inline lib/controller/sfWebController.class.php]: `click` -> `hit`

Comment (caseycs) [inline lib/controller/sfWebController.class.php]: Done

File: lib/controller/sfWebController.class.php (hunk 1 above)
Comment (caseycs) [inline lib/controller/sfWebController.class.php]: Changed

**Pull request merged**